src/ert/storage/local_experiment.py

```diff
@@ -59,11 +60,11 @@ def gen_kw_info(self) -> Dict[str, Any]:
         return priors
 
     @property
-    def field_info(self) -> Dict[str, Any]:
+    def parameter_info(self) -> Dict[str, Any]:
         info: Dict[str, Any]
-        path = self.mount_point / "field-info.json"
+        path = self.mount_point / self._parameter_file
         if not path.exists():
-            raise ValueError("No field info exists")
+            raise ValueError("No parameter.json exists")
```

Comment:
Nit-picking but I think it's better to do `raise ValueError(f"No {str(self._parameter_file)} exists")`